data/tribes/initialization/empire/units.lua

```diff
@@ -1147,13 +1156,23 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_warehouse",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Warehouse
+            lore = pgettext("empire_building",
+               "‘Bis dat, qui cito dat, nil dat, qui munera tardat.’<br>"..
+               "(Who gives promptly, gives twice, who gives late, gives nothing.)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Warehouse
+            lore_author = pgettext("empire_building", "Proverb written over the doors of a warehouse"),
             -- TRANSLATORS: Purpose helptext for an empire warehouse: Warehouse
             purpose = pgettext("empire_building", "Your workers and soldiers will find shelter here. Also stores your wares and tools.")
          }
       },
       {
          name = "empire_port",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Port
+            lore = pgettext("empire_building", "‘Navigare necesse est.’<br>(To sail is necessary.)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Port
+            lore_author = pgettext("empire_building", "Admiral Litus Marius Exsiccatus"),
```

Comment:
who was / is this admiral what has he done to be remembered?   
I believe the lore should explain this for characters not mentioned in the camapigns.  
Suggestion:  victorious leader in the seabattles against the Atlanteans

Reply:
OK, I'll remove the names where possible. (These were meant to be funny if you understand / look up the meaning of the names.)

Reply:
other solution might be to add on which occasion this famous imperial admiral said that. for example Admiral......, at the inauguration of the new Port of Fremil.